src/main/kotlin/mate/academy/Main.kt

```diff
@@ -1,5 +1,21 @@
 package mate.academy
 
+import mate.academy.model.Figure
+import mate.academy.service.ColorSupplier
+import mate.academy.service.FigureSupplier
+
 fun main() {
-    // draw all figures here
+    val colorSupplier = ColorSupplier()
+    val figureSupplier = FigureSupplier(colorSupplier)
+    val numberOfFigures = 6
+    val numberOfRandomFigures = 3
```

Comment:
It is better to attach this value to numberOfFigures. Smth like numberOfFigures / 2, as it should represent the half of figures array